src/components/section-row/section-row.css

```diff
@@ -4,6 +4,10 @@
   display: flex;
 }
 
+.section-row > * {
+  flex-basis: 200px;
```

Comment:
Do we want this for all elements after section row, or does this more pertain to usages with custom fields?

Reply:
As of now, we know we only want "form fields" in section rows and they need to be aligned in a "grid" through spacing.